contentcuration/contentcuration/frontend/shared/views/__tests__/studioBanner.spec.js

```diff
@@ -0,0 +1,54 @@
+import { render, screen } from '@testing-library/vue';
+import VueRouter from 'vue-router';
+import StudioBanner from '../StudioBanner.vue';
+
+// Mock the Kolibri design system composable
+const mockSendPoliteMessage = jest.fn();
+jest.mock('kolibri-design-system/lib/composables/useKLiveRegion', () => ({
+  __esModule: true,
+  default: () => ({
+    sendPoliteMessage: mockSendPoliteMessage,
+  }),
+}));
+
+const sampleErrorMessage = 'This is an error message';
+
+describe('StudioBanner', () => {
+  beforeEach(() => {
+    mockSendPoliteMessage.mockClear();
+  });
+
+  test('render with defaults values', () => {
+    render(StudioBanner, {
+      props: {
+        error: false,
+      },
+      routes: new VueRouter(),
+      slots: {
+        default: '',
+      },
+    });
+
+    const banner = screen.getByTestId('studio-banner');
+    expect(banner).toBeInTheDocument();
+    expect(banner).toHaveClass('banner', 'notranslate');
```

Comment:
There's no need to test for presence of classes - it doesn't really say that much about resulting user experience. In unit tests, it's fine to focus mostly on business logic rather than visual appearance.